pkg/tests/observability_addon_test.go

```diff
@@ -208,7 +208,7 @@ var _ = Describe("Observability:", func() {
 			}, EventuallyTimeoutMinute*5, EventuallyIntervalSecond*5).Should(BeTrue())
 		})
 
-		It("[Integration] Modifying managedcluster cr to enable observability", func() {
+		It("[Stable] Modifying managedcluster cr to enable observability", func() {
```

Comment:
Does it stable? I met this case failure in our PR tests 2 days ago.

Reply:
Got it. let me update this case to Integration.